packages/components/tests/dummy/app/templates/components/pagination.hbs

```diff
@@ -148,157 +365,295 @@
-    <P.Info />
-    <P.Nav />
-    <P.SizeSelector @sizes={{array 10 30 50}} />
-  </Hds::Pagination>
-
-  <h4 class="dummy-h4">“Numbered” pagination with all optional child components</h4>
+  <Hds::Pagination::Numbered @totalItems={{150}} @itemsPerPage={{10}} @pageSizes={{array 10 30 50}} />
+  <p class="dummy-paragraph">If it's necessary it's possible to disable this functionality using a specific
+    <code class="dummy-code">@isTruncated</code>
+    argument:</p>
   {{! prettier-ignore-start }}
   {{! template-lint-disable no-unbalanced-curlies }}
   <CodeBlock
     @language="markup"
-    @code='
-      <Hds::Pagination @totalItems=\{{100}} @itemsPerPage=\{{10}} as |P|>
-        <P.Info />
-        <P.Nav @type="numbered" />
-        <P.SizeSelector @sizes=\{{array 10 30 50}} />
-      </Hds::Pagination>
-    '
+    @code="
+      <Hds::Pagination::Numbered
+        @totalItems=\{{150}}
+        @itemsPerPage=\{{10}}
+        @isTruncated=\{{false}}
+        @pageSizes=\{{array 10 30 50}}
+      />
+    "
   />
   {{! template-lint-enable no-unbalanced-curlies }}
   {{! prettier-ignore-end }}
   <p class="dummy-paragraph">Renders to:</p>
-  <Hds::Pagination @totalItems={{100}} @itemsPerPage={{10}} as |P|>
-    <P.Info />
-    <P.Nav @type="numbered" />
-    <P.SizeSelector @sizes={{array 10 30 50}} />
-  </Hds::Pagination>
+  <Hds::Pagination::Numbered
+    @totalItems={{150}}
+    @itemsPerPage={{10}}
+    @isTruncated={{false}}
+    @pageSizes={{array 10 30 50}}
+  />
 
-  <h4 class="dummy-h4">“Numbered” PaginationNav as a stand alone component</h4>
+  <h5 class="dummy-h5">Events handling</h5>
+  <p class="dummy-paragraph">The component exposes two callback functions that can be used to respond to specific
+    events:</p>
   {{! prettier-ignore-start }}
   {{! template-lint-disable no-unbalanced-curlies }}
   <CodeBlock
     @language="markup"
-    @code='
-      <Hds::Pagination::Nav @totalPages={{10}} @type="numbered" />
-    '
+    @code="
+      <Hds::Pagination::Numbered
+        @totalItems=\{{40}}
+        @itemsPerPage=\{{10}}
+        @pageSizes=\{{array 10 30 50}}
+        @onPageChange=\{{this.handlePageChange}}
+        @onPageSizeChange=\{{this.handlePageSizeChange}}
+      />
+    "
+  />
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::Pagination::Numbered
+    @totalItems={{40}}
+    @itemsPerPage={{10}}
+    @pageSizes={{array 10 30 50}}
+    @onPageChange={{this.handlePageChange}}
+    @onPageSizeChange={{this.handlePageSizeChange}}
+  />
+  <p class="dummy-paragraph">The first function
+    <code class="dummy-code">onPageChange</code>
+    is invoked when a user interacts with a navigation control ("prev/next" or "page number") and so can be used to
+    respond to a "page" change (eg. updating the list of items in the page and/or updating the routing/URL).</p>
+  <p class="dummy-paragraph">The second function
+    <code class="dummy-code">onPageSizeChange</code>
```

Comment:
Similar to above, I would change this to:

"The second function, `onPageSizeChange`, is ..."

Or perhaps to:

"The second `onPageSizeChange` function is ..."

Reply:
Fixed